EssentialsDiscord/src/main/java/net/essentialsx/api/v2/services/discord/InteractionCommandArgument.java

```diff
@@ -0,0 +1,57 @@
+package net.essentialsx.api.v2.services.discord;
+
+/**
+ * Represents an argument for a command to be shown to the discord client.
```

Comment:
```suggestion
 * Represents an argument for a command to be shown to Discord users.
```

`the Discord client` -> `Discord users` because "the client" sounds like you're talking about the client/library we use to connect to Discord